zlib.md

```diff
@@ -0,0 +1,64 @@
+# Performance of ZLIB forks for inflate and deflate
+
+ZLIB is a compression library used to compress and decompress data by popular software such as Java packages, package management using zip, gzip, png etc. Due to its wide usage only rare changes are accepted, which has lead to numerous forks - same interface with improved functionality. Of the many, there are 4 predominantly used zlib forks:
```

Comment:
overall this is too wordy. All we want to say is that for the best perforance on Graviton2 please use the zlib-cloudflare fork and just put it in the generic table we have of libarries.

Reply:
Ok. I'll update the table with an entry and remove zlib.md